### DISCUSSION TITLE
Drag outpost medical items from cabinet to your character's avatar (= using the meds on yourself) is not considered as stealing

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14098] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/FakeFishGames/Barotrauma/assets/64390906/af4d849b-08ed-4f7d-9861-7c31351fe23d

### Reproduction steps

1. go to outpost medical module
2. find some medicines inside cabinets
3. drag them on your character's avatar
4. no reputation loss 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by cl0ck-byte** [comment]
Also on latest versions, ie https://github.com/FakeFishGames/Barotrauma/discussions/17756

This applies to all consumable items and not just medical items.

**Comment by Ivanzyfer** [comment]
Still reproduces on v1.15.19.0 (companion update branch, Home & Harbor 1.13.0.0), single player.

Medicine dragged from an outpost medical cabinet straight onto the character portrait is applied in full and is not counted as stealing. At the end of the round there are no `Applying reputation loss for stealing` lines, and the save has no `reputation.faction.coalition` entry: Coalition reputation stayed at its initial 5. The item never enters the inventory and no NPC reacts.

Checked because v1.15.19.0 added the end-of-round penalty for stolen items brought to the sub, and an item used out of the cabinet is never brought anywhere.

Single player only, and medical items only.
